redisson/src/main/java/org/redisson/api/RSetCacheAsync.java

```diff
@@ -126,6 +126,15 @@ public interface RSetCacheAsync<V> extends RSetAsync<V> {
      * @return amount of added elements
      */
     RFuture<Integer> addAllIfAbsentAsync(Map<V, Duration> objects);
+    /**
+     * Adds elements to this set only if all of them haven't been added before.
+     * <p>
+     * Requires <b>Redis 3.0.2 and higher.</b>
+     *
+     * @param objects map of elements to add
+     * @return amount of added elements
+     */
+    RFuture<Boolean> addAllIfAllAbsentAsync(Map<V, Duration> objects);
```

Comment:
It's better to rename it to `addIfAbsentAsync()`

Reply:
Thanks for reminding.